handler/src/main/java/io/netty/handler/timeout/ReadTimeoutHandler.java

```diff
@@ -95,7 +95,7 @@ protected final void channelIdle(ChannelHandlerContext ctx, IdleStateEvent evt)
      */
     protected void readTimedOut(ChannelHandlerContext ctx) throws Exception {
         if (!closed) {
-            ctx.fireExceptionCaught(ReadTimeoutException.INSTANCE);
```

Comment:
Why wasn't this left as-is? I'm fine with it, but it just didn't follow the pattern of the rest of the PR.

Reply:
Good catch... it’s a left-over . Let me revert